frontend/app/src/components/table-filter/TableFilter.vue

```diff
@@ -82,7 +86,7 @@ function setSearchToMatcherKey(matcher: SearchMatcher<any>) {
   const allowExclusion = 'string' in matcher && matcher.allowExclusion;
   const filter = `${matcher.key}${allowExclusion ? '' : '='}`;
   set(search, filter);
-  get(input).focus();
+  get(input)?.$refs?.textInput?.focus?.();
```

Comment:
Maybe it would make sense to expose a `focus` function from `RuiAutoComplete` to avoid exposing it's internals that much.

Not for this PR but as a follow-up